app/utils/validate_data.py

```diff
@@ -0,0 +1,13 @@
+import json
+from ..middlewares.exceptions import CustomException
+
+def validate_data(request_body, required_fields):
```

Comment:
I believe this to do more than just validate_data. It also specifically validates json body and returns data. I think it needs to be more specific and to also be documented here for its use cases. Document methods you want others to use or make their method names specific enough that it indicates its for very specific use cases.